### PR TITLE
More opinionated edits for the morning chapters

### PR DESCRIPTION
**Pull request by ento**
builds on top of #44

### CODE REVIEW

File: Building User Interfaces.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ Congratulations on completing the last lesson! If this was your first experience
...
 
-Built into the language, Elm has a series of abstractions that are helpful in modeling user interfaces. Together, these abstractions form **the Elm architecture**, which is a detailed template for constructing an application in Elm. Similar to how the strictness of the language pushes you to write code in a particular way, the Elm architecture pushes you to structure applications in a way that prioritizes maintainability.
+As we will see, Elm has a series of abstractions built into the language that are helpful in modeling user interfaces. Together, these abstractions form **the Elm architecture**, which is a detailed template for constructing an application in Elm. Similar to how the strictness of the language pushes you to write code in a particular way, the Elm architecture pushes you to structure applications in a way that prioritizes maintainability.
```
Comment (raorao) [inline Building User Interfaces.md]: nice!

File: Why Elm.md [hunk 2]
```diff
@@ -27,7 +27,7 @@ You may wonder how Elm developers can survive with all these restrictions. The s
...
 
-You may be asking yourself why you should choose to learn a language that's so different from other tools in the ecosystem. **We'd argue that the experience of working in an alien environment can be productive, even if it's just for a little while.**  You are probably not used to dealing with the constraints of purity, immutability, and static typing, which together make up the basic tenets of **functional programming**. These constraints will push you to solve problems in creative ways, and hopefully you will learn strategies that you can employ in other languages.
+You may be asking yourself why you should choose to learn a language that's so different from other tools in the ecosystem. **We'd argue that the experience of working in an alien environment can make you more productive in your home environment, even if the experience is just for a little while.**  You are probably not used to dealing with the constraints of purity, immutability, and static typing, which together make up the basic tenets of **functional programming**. These constraints will push you to solve problems in creative ways, and hopefully you will learn strategies that you can employ in other languages.
```
Comment (raorao) [inline Why Elm.md]: hm, the reference to your "home environment" is a little awkward, and it be may unclear — home could mean "not at work" in this context. What if we kept the original text but changed `productive` to `fruitful`, to make it clear that the gains are unrelated to a particular environment?

File: Why Elm.md [hunk 3]
```diff
@@ -29,6 +29,6 @@ Developing with Elm often feels like an ongoing conversation with the compiler,
 
 You may be asking yourself why you should choose to learn a language that's so different from other tools in the ecosystem. **We'd argue that the experience of working in an alien environment can make you more productive in your home environment, even if the experience is just for a little while.**  You are probably not used to dealing with the constraints of purity, immutability, and static typing, which together make up the basic tenets of **functional programming**. These constraints will push you to solve problems in creative ways, and hopefully you will learn strategies that you can employ in other languages.
 
-We make no claim that functional programming is *better* than other types of programming, but it definitely is *different*. We strongly believe that learning a new way to think about programming will make you a better developer, in the same way that learning a second language makes you appreciate the architecture and beauty of the languages you already know.
```
Comment (raorao) [inline Why Elm.md]: good catch

**Comment by raorao** [comment]
On second thought, this is clearly an improvement over the status quo. approving and merging, I'll push a fix separately.

File: Why Elm.md (hunk 2 above)
Comment (ento) [inline Why Elm.md]: yup, +1 to `fruitful`. 

to explain the intention behind the second half of the change:

```diff
- even if it's just for a little while
+ even if the experience is just for a little while.
```

"it" was a little ambiguous in that it could refer to the productivity boost. rearranging could work too:

> We'd argue that the experience of working in an alien environment, even if it's just for a little while, can be fruitful.

**Comment by ento** [comment]
thanks!

**Comment by raorao** [comment]
love the change. will implement in a sec.
